ansible/vars/icds/icds_public.yml

```diff
@@ -242,7 +242,7 @@ couchdb2:
 
 couch_dbs:
   default:
-    host: 10.247.24.13
+    host: 10.247.24.18
```

Comment:
in the future this should probably be changed to use someting from the inventory instead of an IP